tests/test_iterdictindex.py

```diff
@@ -190,6 +190,31 @@ def test_check_nostemmer_utf_toks(self):
         # restore setting after test
         pt.ApplicationSetup.setProperty("termpipelines", "Stopwords,PorterStemmer")
 
+    def test_check_customstops(self):
+        it = [
+            {'docno': '1', 'url': 'url1', 'text': 'He ran out of money, so he had to stop playing', 'title': 'Woes of playing poker'},
+            {'docno': '2', 'url': 'url2', 'text': 'The waves were crashing on the shore; it was a', 'title': 'Lovely sight'},
+            {'docno': '3', 'url': 'url3', 'text': 'The body may perhaps compensates før the loss', 'title': 'Best of Viktor Prowoll'},
+        ]
+        indexer = pt.IterDictIndexer(self.test_dir, stemmer=pt.TerrierStemmer.none, stopwords=['money', 'crashing'], overwrite=True)
```

Comment:
Can we add a test with UTF characters in the stopwords?

Is there anything that's invalid that should raise an error before causing havoc? Probably `,` and `\n`? Maybe others?

Reply:
i added a fix for comma. whitespace is rarely a token anywhere.